### ISSUE TITLE
add 3d shapes

### ISSUE THREAD

**Issue by DB1ADE** [LibreSolar/kicad-footprints#2]
Hi there,
I have started to design my own standalone configurable BMS for 12V Li Batteries (3S LiIon or 4S LiFePO4).
You did a great job and I tried to copy some of your ideas and libs. The symbols and footprints can easily get imported. But the footprints have no 3d shape.

I started with a Würth screw terminal, but realized, the 3d model has the following path and is not included.
`${KISYS3DMOD}/LibreSolar.3dshapes/Wuerth_WP-THRBU_74650195.stp`

Can you make another repo with the 3d models?
Are there maybe license issues with the models?

**Comment by martinjaeger** [comment]
> Are there maybe license issues with the models?

Exactly. Most vendor 3D models have a proprietary license and I don't want to read the terms and figure out if I'm allowed to publish them. So you will have to download the files from the vendor sides yourself, unfortunately.

**Comment by pasrom** [comment]
@martinjaeger could we document the links for easier download?
